src/styles/calendar.scss

```diff
@@ -35,25 +35,31 @@
     }
   }
 
+  @each $start-day in $week-days {
+    &--start-day--#{$start-day} {
+      .calendar__day:first-child {
+        margin-left: $calendar-width - $square-side;
```

Comment:
this absolutely will not working during the task, you can test it by adding to calendar block class calender--start-day--fri for example

Reply:
dont know how fix it can u assist???????????